### PR TITLE
add service definition for form field title extractor

### PR DESCRIPTION
**Pull request by Guite**
This PR adds a service definition for the extractor wanted in https://github.com/php-translation/symfony-bundle/issues/394

Do not merge before https://github.com/php-translation/extractor/pull/148 is merged.

### CODE REVIEW

File: Resources/config/extractors.yaml [hunk 1]
```diff
@@ -49,6 +49,10 @@ services:
     tags:
       - { name: 'php_translation.visitor', type: 'php' }
 
+  Translation\Extractor\Visitor\Php\Symfony\FormTypeTitle:
```
Comment (rvanlaak) [inline Resources/config/extractors.yaml]: What about solving this for all future extractors / visitors:

```
_instanceof:
    PhpParser\NodeVisitor:
        tags:
            - { name: 'php_translation.visitor', type: 'php' }
```

Comment (Guite) [reply Resources/config/extractors.yaml]: Shouldn't this be done in a separate step?

Comment (odolbeau) [reply Resources/config/extractors.yaml]: As you wish. :)
I'm releasing a new version of the extractor soon, we can merge this PR once done & @rvanlaak you can create a new PR with your proposal?

**Comment by odolbeau** [comment]
Version 2.0.1 of extractor released!
https://github.com/php-translation/extractor/releases/tag/2.0.1

File: Resources/config/extractors.yaml (hunk 1 above)
Comment (rvanlaak) [reply Resources/config/extractors.yaml]: Feel free to merge, I'll experiment in a separate PR 👍

Comment (rvanlaak) [reply Resources/config/extractors.yaml]: Created PR #396 to simplify service configuration

**Comment by Guite** [comment]
what's the state of this PR? anything to do in order to get it merged?

**Comment by odolbeau** [comment]
Could you remove the non-trigerred PHPStan error & replace the service declaration with the code suggested by @rvanlaak please?
```
_instanceof:
    PhpParser\NodeVisitor:
        tags:
            - { name: 'php_translation.visitor', type: 'php' }
```
It will avoid the current Travis error when the bundle is used with a `php-translation/extractor` version which doesn't contain the new visitor.

It's done in #396 but this PR does several things. :/

**Comment by Guite** [comment]
@odolbeau done both but eventually this lead to including almost everything from #396 - except the autowire enablement.

**Comment by odolbeau** [comment]
@Guite could you squash your commits please?
@rvanlaak WDYT?
It it OK for you if we merge this PR which is now ready to be merged or do you prefer to correct #396?

**Comment by Guite** [comment]
> could you squash your commits please?

Please do so when clicking on the merge button.

**Pull request merged**